File: leetcode/Accepted/kids-with-the-greatest-number-of-candies.cpp
class Solution {
  public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
      int mx = *max_element(candies.begin(), candies.end());
      vector<bool> res;
      for(int item : candies){
        res.push_back(item+extraCandies >= mx? true : false);
      }
      return res;
    }
};

File: leetcode/AC/Easy/kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        mx = max(candies)
        res = []
        for item in candies:
            res.append(True if item + extraCandies >= mx else False)

        return res
